fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/SessionFactory.java

```diff
@@ -136,6 +136,16 @@ public Session getSession(final HttpServletRequest servletRequest) {
     }
 
     /**
+     * Determine whether the given Session is part of a transaction
+     *
+     * @param session
+     * @return whether the session is part of a transaction
+     */
+    public boolean isInTransaction(final Session session) {
```

Comment:
Are we trying to get JCR out of `-http-commons`? Is this is a temporary maneuver?